code-env/python/spec/requirements.txt

```diff
@@ -1,4 +1,4 @@
-mxnet==1.7.0.post1
+mxnet-cu100==1.7.0
```

Comment:
This will be changed by the make rule for gpu, can you leave the previous version? The tests will need to change this manually.